### PR TITLE
app-text/sigil: bump v1.0.0

### PR DESCRIPTION
**Pull request by arthurzam**
The new version `1.0.0` was released today.
- use new `PYTHON_MULTI_USEDEP` API - Please check I have done it correctly.
- remove unneeded `dev-libs/boost` and `dev-libs/xerces-c` dependency - happened some releases before, but I have missed it - fixed now.
- remove `readme.gentoo` file as the notice is no longer needed, it was fixed in this release.

Package-Manager: Portage-2.3.82, Repoman-2.3.2

### CODE REVIEW

File: app-text/sigil/sigil-1.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		dev-python/lxml[${PYTHON_MULTI_USEDEP}]
+		dev-python/six[${PYTHON_MULTI_USEDEP}]')
+	>=dev-qt/qtconcurrent-5.12:5
+	>=dev-qt/qtcore-5.12:5
+	>=dev-qt/qtgui-5.12:5
+	>=dev-qt/qtprintsupport-5.12:5
+	>=dev-qt/qtwebengine-5.12:5[widgets]
+	>=dev-qt/qtwidgets-5.12:5
+	sys-libs/zlib[minizip]
+	plugins? ( $(python_gen_cond_dep \
```
Comment (juippis) [inline app-text/sigil/sigil-1.0.0.ebuild]: That's how I've also understood how to use python-single-r1 now, but I don't see function for this USE flag in this ebuild?
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs

File: app-text/sigil/sigil-1.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
```
Comment (juippis) [inline app-text/sigil/sigil-1.0.0.ebuild]: py3.5 support will be dropped soon, you can still leave it, but you should start testing py3.8 support on these contributions if possible.

Comment (arthurzam) [reply app-text/sigil/sigil-1.0.0.ebuild]: I prefer that py3.5 support is removed with all other python packages.
For py3.8 support I need `dev-python/cssutils` and `dev-python/css-parser` to support py3.8.

File: app-text/sigil/sigil-1.0.0.ebuild (hunk 1 above)
Comment (arthurzam) [reply app-text/sigil/sigil-1.0.0.ebuild]: Thanks for the reply.
If I understand correctly the example given there, (I will quote):
`... except under very specific circumstances, such as a package being nonfunctional unless at least one of a set of RDEPs is installed ...`
All those packages are needed for plugins support and I think a USE flag is a cleaner version.

Comment (juippis) [reply app-text/sigil/sigil-1.0.0.ebuild]: I always decoded it as IUSE="engine1 engine2", and `optfeature` can print multiple packages per feature, but yes I agree this is much cleaner.

**Pull request closed without merging**